src/client/containers/layout.js

```diff
@@ -198,6 +222,8 @@ class Layout extends React.Component {
 			>
 				{checkPrivilege(user.privs, PrivilegeType.ADMIN) && adminOptions}
 				{checkPrivilege(user.privs, PrivilegeType.RELATIONSHIP_TYPE_EDITOR) && relationshipTypeEditorOptions}
+				{checkPrivilege(user.privs, PrivilegeType.IDENTIFIER_TYPE_EDITOR) && identifierTypeEditorOptions}
+				{checkPrivilege(user.privs, PrivilegeType.REINDEX_SEARCH_SERVER) && reindexSearchEngineOption}
```

Comment:
Despite having the correct privileges on test.BB, I don't see the reindex search server option in the privileges menu:
<img width="255" alt="image" src="https://github.com/metabrainz/bookbrainz-site/assets/6179856/86d5e280-0b32-40de-b113-22b4ba6ea328">

What do?

I can however access the /search/reindex route successfully with the right privs, but not without the right privs so on that front everything works as expected :+1:

Reply:
~~I looked into it a bit more and found the underlying issue: the checkPrivilege function returns false for the inputs 23 and 8 (23 being my current privs and 8 being the REINDEX_SEARCH_SERVER priv)~~

~~If you try the JS snippet `Boolean(23 & 8)` indeed it evaluates as false~~



THAT'S RIGHT! I forgot I need to log out and log back in to have my privs up to date on the session.
Will definitely have to think about how to resolve that issue (busting the session cache for that user or forcefully logging them out), but it's not a blocker.